themes/inbeat/layouts/partials/footer/scripts.html

```diff
@@ -1,5 +1,5 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/bodymovin/5.6.8/lottie.min.js" integrity="sha256-HlqBFI4b3mzFNE97JQcgI/uhJvHBI5czBqfVtEeA1Hg=" crossorigin="anonymous"></script>
-<script src="/js/main.min.js"></script>
+<script src="{{ "/js/main.min.js" | absURL }}" defer></script>
```

Comment:
Are you sure the `defer` is OK? Works well deferred?

Reply:
It should! I'll just revert if it's not working well